patches/minecraft/net/minecraft/world/entity/item/ItemEntity.java.patch

```diff
@@ -104,11 +104,12 @@
  
        if (p_32034_.m_128403_("Owner")) {
           this.f_31982_ = p_32034_.m_128342_("Owner");
-@@ -303,10 +_,18 @@
+@@ -303,10 +_,19 @@
  
     public void m_6123_(Player p_32040_) {
        if (!this.f_19853_.f_46443_) {
 +         if (this.f_31986_ > 0) return;
++         if(!isPickupAllowed(p_32040_, net.minecraftforge.items.pickup.ItemPickupReasons.PLAYER)) return; // Forge: Check to see if item currently disallows being picked up by players
```

Comment:
Formatting: space between `if` and the condition. Additionally, add a `the` between `if` and `item` in the comment.